_posts/0000-01-02-nidakouser.md.

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"I feel a love, I have never felt before, when in prayer, my head hits the floor."